docs/clone.md

```diff
@@ -0,0 +1,30 @@
+# Clone
+
+## Using `clone`
+
+```javascript
+import clone from 'consus-core/clone';
+
+let original = {
+    a: [0, 1, 2],
+    b: 'Hello, World!',
+    c: 3.14159265
+};
+
+let twin = clone(original);
+
+twin;
+/*
+{
+    a: [0, 1, 2],
+    b: 'Hello, World!',
+    c: 3.14159265
+}
+*/
+
+twin == original;
+// false
+
+twin.a == original.a;
```

Comment:
I'm not able to find a scenario where `===` differs from `==` when comparing objects, but I'll switch this since I always use `===` elsewhere.